enterprise/users/src/main/java/io/crate/plugin/EnterpriseUsersExtension.java

```diff
@@ -41,25 +41,25 @@
     public List<NamedWriteableRegistry.Entry> getNamedWriteables() {
         List<NamedWriteableRegistry.Entry> entries = new ArrayList<>(4);
         entries.add(new NamedWriteableRegistry.Entry(
-            MetaData.Custom.class,
-            UsersMetaData.TYPE,
-            UsersMetaData::new
+                Metadata.Custom.class,
+                UsersMetadata.TYPE,
+                UsersMetadata::new
         ));
         entries.add(new NamedWriteableRegistry.Entry(
-            NamedDiff.class,
-            UsersMetaData.TYPE,
-            in -> UsersMetaData.readDiffFrom(MetaData.Custom.class, UsersMetaData.TYPE, in)
+                NamedDiff.class,
```

Comment:
Why the additional indentation changes everywhere? This diverges from our usual style

Reply:
My bad, this got introduced by intellij using the refactoring feature before i switched to sed. However, i reverted the formatting globally now. Sorry for that.